core/o11y/src/lib.rs

```diff
@@ -118,6 +174,13 @@ impl<'a> EnvFilterBuilder<'a> {
         Self { rust_log: rust_log.into(), verbose: None }
     }
 
+    /// Create the `EnvFilter` from the given logging directives or the [`DEFAULT_RUST_LOG`] value if no directives are given
+    ///
+    /// This method will not inspect the environment variable.
+    pub fn from_default() -> Self {
```

Comment:
This should be `Default::default` implementation rather than a `from_deafault` method.